doc/src/classes/helpers.rst

```diff
@@ -45,7 +45,7 @@ how to use these types.
 .. cpp:class:: template <class T> chemfiles::span
 
     A ``span<T>`` is a view inside a ``std::vector<T>`` providing all the
-    operations of a vector expect for memory allocation. The idea and
+    operations of a vector excepted for memory allocation. The idea and
```

Comment:
As a non-native speaker hard to say, but does "... except for ..." sound better? So all allocations would be excepted (passive). =)

Reply:
non-native speaker here too, but yes, this sounds nicer!